### DISCUSSION TITLE
how to debug when ui.open goes corrected anchor but place it under header?

### DISCUSSION THREAD

**Discussion by retsyo** [zauberzeug/nicegui#2115] [Q&A]
### Question

first of all, I am very sorry for no minimal reproducible code because when I try to strip down my real code, I find that the stripped one works as expected.

The phenomenon is, in my real project, I placed some `ui.link_target`s in the beginnings of of many `ui.html`'s in most of the remain page space, and some `ui.button`s in the header which has a function to jump to every anchors respectively. When my project runs in webbrowser, I can read something like

```html
<a id="c9" name="quiz_01"></a>
``` 

if I press F12 and select the item

The button can fire something

```py
http://127.0.0.1:8080/#quiz_01
```

and I can notice there is a navigation to item whose name is `quiz_01`. But this item appears on the left corner of webbrowser window, in other word, user can not read the content since it is covered by header. 

Any hints to fix it?

Sorry again for I can't find out a minimal reproducible code currently.

**Comment by falkoschindler** [comment]
Sorry, a minimal reproducible example would be *really* helpful to understand the problem. Right now I'm not sure I understand what is happening.
